tests/e2e/yaml/secured-cluster-cr.envsubst.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: platform.stackrox.io/v1alpha1
```

Comment:
Do you mind deleting `tests/e2e/yaml/secured-cluster-cr.yaml` file if it's no longer used?

Reply:
Done